packages/clima_ui/lib/screens/location_screen.dart

```diff
@@ -76,11 +74,26 @@ class _LocationScreenState extends State<LocationScreen> {
     final _themeState = context.read(themeStateNotifier);
     final weatherState = useProvider(w.weatherStateNotifierProvider.state);
     final weatherStateNotifier = useProvider(w.weatherStateNotifierProvider);
+    final cityName = useState('');
+    final FloatingSearchBarController controller =
+        FloatingSearchBarController();
 
     final isLoading = useState(weatherState is c.Loading);
 
     final cityStateNotifier = useProvider(c.cityStateNotifierProvider);
 
+    // @override
+    // void initState() {
+    //   super.initState();
+    //   controller = FloatingSearchBarController();
+    // }
+    //
+    // @override
+    // void dispose() {
+    //   controller.dispose();
+    //   super.dispose();
+    // }
+
```

Comment:
All of this should be deleted. You should generally avoid committing commented-out code.